### PR TITLE
CANDLEPIN-519: Switch to main branch

### PR DESCRIPTION
**Pull request by Januson**

### CODE REVIEW

File: src/main/groovy/com/dslexample/builder/GradleCiJobBuilder.groovy [hunk 1]
```diff
@@ -14,7 +14,7 @@ class GradleCiJobBuilder {
   String name
   String description
   String ownerAndProject
-  String gitBranch = 'master'
```
Comment (nikosmoum) [inline src/main/groovy/com/dslexample/builder/GradleCiJobBuilder.groovy:17]: Is this change needed? where is this script used exactly?

Comment (Januson) [reply src/main/groovy/com/dslexample/builder/GradleCiJobBuilder.groovy:17]: It's not necessary. This class is an example, but I changed it as well to not leave unchanged pieces behind.
